### PR TITLE
APR-1990 : Two in Twelve Months

### PR DESCRIPTION
**Pull request by martin-hornsby**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using SFA.DAS.ApplyService.Domain.Apply;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ApplyService.Web.ViewModels.Roatp
+{
+    public class TwoInTwelveMonthsViewModel : IPageViewModel
+    {
+        public bool? HasTwoInTwelveMonths { get; set; }
+
+        public string Title { get { return "Have you had 2 unsuccessful applications in the last 12 months?"; } set { } }
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs:11]: does this need a set? If not, => will do

File: src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        public string GetHelpQuestion { get; set; }
+
+        public bool GetHelpQuerySubmitted { get; set; }
+
+        public string GetHelpErrorMessage { get; set; }
+
+        public List<ValidationErrorDetail> ErrorMessages { get; set; }
+
+        public string GetHelpAction { get { return "TwoInTwelveMonths"; } set { } }
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs:27]: Does this need a set?

**Review by MarkFCain — DISMISSED** [review]
some minor comments that don't really matter.

File: src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs:11]: abstract base class so needs an implementation :rage1:

File: src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TwoInTwelveMonthsViewModel.cs:27]: abstract base class so needs an implementation :rage1:

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**